jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -51,6 +51,11 @@ def vue_calculate_moment(self, *args):
         # Retrieve the data cube and slice out desired region, if specified
         cube = self.dataset.get_object(cls=Spectrum1D, statistic=None)
         spec_min, spec_max = self.spectral_subset.selected_min_max(cube)
+        if spec_min.target is None:
+            spec_min.target = spec_min.observer
+        if spec_max.target is None:
+            spec_max.target = spec_max.observer
```

Comment:
can/should this be moved into the `selected_min_max` method?  That would cut us down from needing this in three instances to two (I would love to see it moved even further upstream to get it to one if possible, but that probably isn't worth it unless/until we confirm this is the logic/fix we need).